tests/suites/cloud_gce/task.sh

```diff
@@ -23,5 +23,7 @@ test_cloud_gce() {
 
 	test_deploy_gpu_instance
 
+	test_attach_storage_pool
```

Comment:
We're not attaching a storage pool.
We can just call it `test_storage_pool`

Reply:
Fixed it.